README.md

```diff
@@ -1,11 +1,12 @@
-# cron <<Experimental>>
+# cron (Experimental)
+
 Cron is a dojot's microservice that allows you to schedule events to be emitted to other microservices.
 
 This service is under development, and new commits are comming soon.
 
-# TODO
+## TODO
 
-- Persist jobs' configuration.
-- Handle cron's restarts.
+- Change to use dojot's logging library
```

Comment:
Is missing some point? I've seen several uses of dojot's logger during this PR.

Reply:
In the commit when I added this TODO item, the cron service wasn't using the logging library. But I forgot to remove it when later I changed to use the library.